packages/core/admin/admin/src/contexts/index.js

```diff
@@ -1,4 +1,4 @@
 export { default as AdminContext } from './Admin';
 export { default as ConfigurationsContext } from './Configurations';
-export { default as PermissionsDataManagerContext } from './PermisssionsDataManagerContext';
+export { default as PermissionsDataManagerContext } from './PermissionsDataManagerContext';
```

Comment:
More renames. Note that this is another case where the local name didn't match the filename.